timescale-container/test/ql-db-init/mtlusovini.etdevice.csv

```diff
@@ -0,0 +1,2080 @@
+accumulatedprecipitationlevel24,airhumidity,airpressure,airtemperature,batterylevel,entity_id,entity_type,fiware_servicepath,latitude,leafweatness,location,location_centroid,longitude,manufacturername,precipitationlevel,previousprecipitationlevel,soilmoisture450,soilmoisture800,soiltemperature,solarradiation,time_index,timeinstant,winddirection,windspeed

```

Comment:
i am not sure i would refer to lusovini in a public repo.

Reply:
got rid of it, thx!